src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -124,6 +124,10 @@ def __init__(self, config):
         if config.rotary_dim is not None:
             self.rotary_dim = config.rotary_dim
 
+        sin, cos = fixed_pos_embedding(x=torch.zeros(1, 1, self.rotary_dim), seq_len=max_positions)
+        self.register_buffer("sin", sin, persistent=False)
+        self.register_buffer("cos", cos, persistent=False)
```

Comment:
Since you pass `seq_len=max_positions` to `fixed_pos_embedding`, the registered `sin` and `cos` should have seq. dim equal to `max_positions`. This changes the shapes of `sin` and `cos` before and after this PR.

This doesn't cause any issue?